### PR TITLE
Implement pwd builtin

### PR DESCRIPTION
**Pull request by delthas**
- implements pwd builtin
- set PWD on startup

### CODE REVIEW

File: builtin/pwd.c [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+#include <stdlib.h>
+#include <mrsh/shell.h>
+#include "builtin.h"
+
+// usage: pwd [-L|-P]
+
+int builtin_pwd(struct mrsh_state *state, int argc, char *argv[]) {
+  // TODO `-P` and `-L`
+  const char *pwd = mrsh_env_get(state, "PWD", NULL);
+  puts(pwd);
```
Comment (emersion) [inline builtin/pwd.c]: This should have a NULL check, in case the user `unset`s it.

File: main.c [hunk 2]
```diff
@@ -87,8 +87,10 @@ int main(int argc, char *argv[]) {
 	if (state.interactive && !(state.options & MRSH_OPT_NOEXEC)) {
 		source_profile(&state);
 	}
-
-	// TODO: set PWD
+	
+	char cwd[PATH_MAX];
+	getcwd(cwd, PATH_MAX);
+	mrsh_env_set(&state, "PWD", cwd, MRSH_VAR_ATTRIB_EXPORT);
```
Comment (emersion) [inline main.c]: Setting PWD is a little bit complicated: see http://pubs.opengroup.org/onlinepubs/9699919799/utilities/V3_chap02.html#tag_18_05_03

You can just implement part of it and leave the rest as a TODO if necessary.

File: builtin/pwd.c (hunk 1 above)
Comment (delthas) [reply builtin/pwd.c]: Done

File: main.c (hunk 2 above)
Comment (delthas) [reply main.c]: Done

File: builtin/pwd.c [hunk 3]
```diff
@@ -0,0 +1,19 @@
+#define _POSIX_C_SOURCE 200809L
+#include <stdio.h>
+#include <stdlib.h>
+#include <mrsh/shell.h>
+#include "builtin.h"
+
+// usage: pwd [-L|-P]
+
+int builtin_pwd(struct mrsh_state *state, int argc, char *argv[]) {
+  // TODO `-P` and `-L`
```
Comment (emersion) [inline builtin/pwd.c]: We use tabs for intendation

File: main.c [hunk 4]
```diff
@@ -87,8 +87,14 @@ int main(int argc, char *argv[]) {
 	if (state.interactive && !(state.options & MRSH_OPT_NOEXEC)) {
 		source_profile(&state);
 	}
-
-	// TODO: set PWD
+	
+	// TODO check if path is well-formed, has . or .., and handle symbolic links
+	const char *pwd = mrsh_env_get(&state, "PWD", NULL);
+	if(pwd == NULL || strlen(pwd) >= PATH_MAX) {
```
Comment (emersion) [inline main.c]: Style: missing space between `if` and `(`

**Comment by emersion** [comment]
Thanks!

**Pull request merged**